.devcontainer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+## ================================================================================================
+# Utility versions
+## ================================================================================================
+ARG KUBECTL_VERSION=1.31.0
```

Comment:
I think `renovate` won't work here? why not just put them in `FROM` so `renovate will update them? I think you do this for better viewability? But not having `renovate` to handle these packages seems not that worth it for me tbh. Or maybe there's other reason for this?

Reply:
I was pleasantly surprised by this too, but renovate does indeed pick those up.

Since I found that out, this is the way I've been doing my devcontainers, and it's picking them up nicely

It will automatically concatenate the arg as the tag to the image. It should also trigger a run in which it will pin these to their digest

Reply:
Wow that's nice to know, thanks!

Reply:
Oh yeah it works! https://github.com/budimanjojo/talhelper/pull/592